.devcontainer/Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN echo "Install general purpose packages" && \
         gdb \
         lcov \
         libclang-11-dev \
-        lldb \
+        lldb-11 \
```

Comment:
`lldb` means `lldb-10`. Given that all the other packages here depend on `llvm-11` and this one on `llvm-10`, I assumed without testing that we can use `lldb-11` here.